.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: Node.js CI
+
+on:
+  schedule:
+    - cron:  '23 4 * * *' # Once, nightly
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 15, 14, 12, 10 ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm run tests-only
```

Comment:
IMO, a lint failure is a test failure.  Is there a reason not to also fail on lint errors and so just run `npm t` in all of them?  I have in the past seen the lint configuration fail in a certain version of node, and if that is the case I think we would want to know.

Reply:
I don't really have strong opinion here, from my side both variants are OK, so if you'd like to remove separate lint job and run lint on every node version, please let me know.

Reply:
It keeps it simple, but I would be interested in hearing others opinions on the topic.  Lets not consider this a blocker, and if no one shares an opinion we can merge and revisit later.